CHANGELOG.md

```diff
@@ -44,6 +44,9 @@
 * The `generate` section of the config now has a `teacher` section. This section configures the
   teacher model when it is automatically served in the background. This new section has the same
   values as the `serve` section of the config.
+* Legacy Linux training now supports the new messages format. When a dataset is provided in the
+  HuggingFace messages format, `ilab` will automatically convert it back into the legacy format.
+* Legacy Linux training is now compatible with the phase007 pretraining format.
```

Comment:
phase007? Where can I read more about that? :)

Reply:
We don't have any documentation up for phase007 yet as I believe this is still something being worked on by SDG and training. But you can read about how it's used by the full training library in https://github.com/instructlab/training/pull/137